index.html

```diff
@@ -20,28 +20,28 @@
           <div class="mdl-cell mdl-cell--2-col mdl-cell--hide-tablet mdl-cell--hide-phone"></div>
           <div class="demo-content mdl-color--white mdl-shadow--4dp content mdl-color-text--grey-800 mdl-cell mdl-cell--8-col">
               <div class="mdl-layout__header-row">
-                <span class="mdl-layout-title">Gerador de Assinaturas da Tegra</span>
+                <span class="mdl-layout-title">Tegra's Signature Generator</span>
               </div>
 
             <form action="#">
               <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
                 <input class="mdl-textfield__input" id="name" type="text" ng-model="$ctrl.model.name">
-                <label class="mdl-textfield__label" for="name">Nome</label>
+                <label class="mdl-textfield__label" for="name">Name</label>
               </div>
               <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
                 <input class="mdl-textfield__input" id="cellphone" type="text" mask="(99) 99999?-9999" mask-validate="false" ng-model="$ctrl.model.cellphone">
-                <label class="mdl-textfield__label" for="cellphone">Celular</label>
+                <label class="mdl-textfield__label" for="cellphone">Cell Phone</label>
```

Comment:
It's better to use "Mobile Number" instead of "Cell Phone" since it's a more common implementation.

![facebookmobilenumber](https://cloud.githubusercontent.com/assets/11710288/21183574/b65f0852-c1ef-11e6-8a6a-d337eb910ac1.png)